### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by hub-bla**

### CODE REVIEW

File: client/src/components/main/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './Main';
```
Comment (Safeheaded) [inline client/src/components/main/index.ts:1]: Add new line at the end of the file (soon we'll have .editorconfig, which automatically does that).

File: client/src/pages/MainPage/MainPage.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Main } from '../../components/main';
+
+// needed:
+// separate component with all
```
Comment (Safeheaded) [inline client/src/pages/MainPage/MainPage.tsx:4]: What do you mean? Separate component with all what?

**Review by Safeheaded — CHANGES_REQUESTED** [review]

File: client/src/pages/MainPage/MainPage.tsx (hunk 2 above)
Comment (hub-bla) [reply client/src/pages/MainPage/MainPage.tsx:4]: It was my side note to create separate component with things that will collect data in the future. Basically just for readability.

File: client/src/components/main/index.ts (hunk 1 above)
Comment (hub-bla) [reply client/src/components/main/index.ts]: Done!

**Review by Safeheaded — APPROVED** [review]

**Pull request merged**